source/blog/2018-09-14-the-ember-times-issue-64.md

```diff
@@ -31,8 +31,19 @@ Curious? [**Read the full proposal**](https://github.com/emberjs/rfcs/pull/372)
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [State of JavaScript 2018 Survey Is Here Again 🗳️](https://medium.freecodecamp.org/take-the-state-of-javascript-2018-survey-c43be2fcaa9)
+
+"Just like Christmas or the flu, the **State of JavaScript** survey comes back around every year. But unlike these, it’s something to actually look forward to!" - [Sacha Greif](https://twitter.com/SachaGreif/status/1037603748917403648)
```

Comment:
Awesome writeup! What do you think about mentioning something non-religious like "New Year's Eve" instead of Christmas to be more inclusive?

Reply:
While I agree, being a non Christmas celebrating person myself, this is a direct quote from the Medium article 😅 

Still makes sense to change and paraphrase?

Reply:
I agree with Amy on this one - if you like you can also just link to the survey directly without any reference to the freecodecamp article and let your creative spirit flow regarding the intro :sparkles:

Reply:
All right :)